packages/web-react/src/components/TextFieldBase/README.md

```diff
@@ -43,24 +43,25 @@ TextFieldBase with password toggle (button to reveal the password):
-| `pattern`           | `string`                                                      | -       | no       | Defines regular expressions for allowed value types                     |
-| `placeholder`       | `string`                                                      | -       | no       | Input placeholder                                                       |
-| `ref`               | `ForwardedRef<HTMLInputElement or HTMLTextAreaElement>`       | -       | no       | Field element reference                                                 |
-| `type`              | `email`, `number`, `password`, `search`, `tel`, `text`, `url` | -       | no       | Input type                                                              |
-| `validationState`   | [Validation dictionary][dictionary-validation]                | -       | no       | Type of validation state.                                               |
-| `validationText`    | `string`, `string[]`                                          | -       | no       | Validation text                                                         |
-| `value`             | `string`                                                      | -       | no       | Input value                                                             |
+| `autoComplete`      | `string`                                                      | –       | no       | [Automated assistance in filling][autocomplete-attr]                    |
+| `hasPasswordToggle` | `boolean`                                                     | –       | no       | If true, the `type` is set to `password` and a password toggle is shown |
+| `helperText`        | `string`                                                      | –       | no       | Custom helper text                                                      |
+| `id`                | `string`                                                      | –       | yes      | Input and label identification                                          |
+| `isDisabled`        | `boolean`                                                     | –       | no       | Whether is field disabled                                               |
+| `isLabelHidden`     | `boolean`                                                     | –       | no       | Whether is label hidden                                                 |
+| `isMultiline`       | `boolean`                                                     | –       | no       | Whether is DOM element `textarea`                                       |
+| `isRequired`        | `boolean`                                                     | –       | no       | Whether is field required                                               |
+| `label`             | `string`                                                      | –       | yes      | Label text                                                              |
```

Comment:
I am afraid that the label is not technically required by the types.

Reply:
[If I understand you well](https://github.com/lmc-eu/spirit-design-system/pull/957/commits/8540af8929b40e8c5c6bafda21ee1b4c6647ff5b#r1276716966), currently it's not possible define `label` as required prop in the React? Why?

Reply:
It is technically possible, but it is a BC, IMHO.

Reply:
Ah, you're right. I reverted those changes, so we/you could fix this in another PR.

Reply:
Thanks :-)